src/specs/components/timeline/demo/data.js

```diff
@@ -8,100 +8,73 @@ module.exports = {
     from: 5,
     to: -2,
   },
-  button: {
```

Comment:
I see that in the timeline template we define a _toggler variable, we set that initially to false and when we have to render the toggle we switch it to true. The question is, why..? :) It seems this is totally useless